src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -41,6 +42,9 @@ public class DatabaseChanges : IDatabaseChanges
 
         private readonly ConcurrentDictionary<DatabaseChangesOptions, DatabaseConnectionState> _counters = new ConcurrentDictionary<DatabaseChangesOptions, DatabaseConnectionState>();
         private int _immediateConnection;
+        
+        private readonly TaskCompletionSource<ChangesSupportedFeatures> _supportedFeaturesTcs = new();
+        public Task<ChangesSupportedFeatures> SupportedFeatures => _supportedFeaturesTcs.Task;
```

Comment:
internal?

Reply:
I think it would be better maybe to mark this as private and do:

```
internal ChangesSupportedFeatures GetSupportedFeaturesAsync() => _supportedFeaturesTcs.Task;
```

Reply:
Done